### ISSUE TITLE
Error merge apk 

### ISSUE THREAD

**Issue by dfdfdfs** [FoelliX/AMT#3]
Hello 
Error 

SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Exception in thread "main" soot.AndroidPlatformException: Android platform directory 'C:\Androidplatforms' does not exist!
        at soot.Scene.getAndroidAPIVersion(Scene.java:392)
        at soot.Scene.getAndroidJarPath(Scene.java:362)
        at soot.Scene.defaultAndroidClassPath(Scene.java:621)
        at soot.Scene.defaultClassPath(Scene.java:560)
        at soot.Scene.getSootClassPath(Scene.java:294)
        at soot.SourceLocator.getClassSource(SourceLocator.java:168)
        at soot.Scene.tryLoadClass(Scene.java:900)
        at soot.Scene.loadBasicClasses(Scene.java:1652)
        at soot.Scene.loadNecessaryClasses(Scene.java:1738)
        at de.upb.mike.amt.soot.SootObject.parseApp(SootObject.java:83)
        at de.upb.mike.amt.AMT.start(AMT.java:157)
        at de.upb.mike.amt.AMT.main(AMT.java:109)

**Comment by FoelliX** [comment]
Hi!

From the error log it seems like you did not declare a valid android platforms directory in the amt.properties file.

"Android platform directory 'C:\Androidplatforms' does not exist!"

Make sure the folder exists and contains the android platform files (e.g. https://github.com/Sable/android-platforms or just the one from the android sdk).

Cheers,
FoelliX

**Comment by dfdfdfs** [comment]
![686866](https://user-images.githubusercontent.com/24856424/123544563-91528700-d75c-11eb-956a-581fbec06976.PNG)

**Comment by dfdfdfs** [comment]
Error

**Comment by FoelliX** [comment]
Hi again!

1. Please use the platforms folder (not android-23).

2. In .properties files you have to escape the backslash. Instead of "\" use "\\" or "/".

3. Your apktool path seems to be wrong as well. Please provide one to an instance of apktool (https://github.com/iBotPeaches/Apktool).

Cheers,
FoelliX

**Comment by dfdfdfs** [comment]
[Error.txt](https://github.com/FoelliX/AMT/files/6721600/Error.txt)
![gtttt](https://user-images.githubusercontent.com/24856424/123545189-8816e980-d75f-11eb-8ca8-39244dba719d.PNG)
No classes.dex

**Comment by FoelliX** [comment]
1. There is no AQL config. Thus, I assume you only want to combine the apps. In this case use
"java -jar AMT-0.0.4.jar A.apk B.apk"
only.

2. Something fails with invalid code messages. Cannot tell what the problem is. You could send me the apks and I can check next week asap.

**Comment by dfdfdfs** [comment]
Okay Thanks.

java -jar AMT-0.0.4.jar A.apk B.apk      Error 
[ERROR  java -jar AMT-0.0.4.jar A.apk B.apk.txt](https://github.com/FoelliX/AMT/files/6721640/ERROR.java.-jar.AMT-0.0.4.jar.A.apk.B.apk.txt)
[A AND B.zip](https://github.com/FoelliX/AMT/files/6721644/A.AND.B.zip)

**Comment by dfdfdfs** [comment]
Hi, have you watched the apk?

**Comment by FoelliX** [comment]
Not yet, but my vacation ends tomorrow. I will do it asap and reply to this issue again.

**Comment by FoelliX** [comment]
Hi again!

In Line 66 of https://github.com/FoelliX/AMT/blob/master/src/de/upb/mike/amt/soot/SootObject.java some packages are set that will be ignored while merging. When I add `kotlin.*` the apps can be merged - no deeper insights, yet. However, ApkTool cannot merge the two manifests since there are some layout variables missing - no layout files or similar are currently merged by AMT. Nonetheless, the output (without merged manifests) gets generated. Effectively only the source code is merged.

I added an option to the `amt.properties` file to setup custom excludes (such as `kotlin.*`). However, I cannot release this new version of AMT yet, since this new version relies on yet un-released other projects.

Nonetheless, I attached a version of AMT with `kotlin.*` added to the list of excludes ([AMT-0.0.4-adapted.zip](https://github.com/FoelliX/AMT/files/6771558/AMT-0.0.4-adapted.zip)).
I also attached the merged apk ([mergedApk.zip](https://github.com/FoelliX/AMT/files/6771555/mergedApk.zip)) and the associated log ([log.txt](https://github.com/FoelliX/AMT/files/6771548/log.txt)).

Cheers,
FoelliX
